don't install eslint-plugin-standard

### What happened?

I asked for standard linting, and it took me down a path which resuted in errors
specifically a conflicting dependency (`eslint-plugin-n`)

Reading around I find that `eslint-plugin-standard` is deprecated - not needed - any more https://github.com/standard/standard/issues/1316



### What did you expect to happen?

standard config / deps that "just work"

### Reproduction URL

-

### How to reproduce?

-

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite)

### Platforms/Browsers

_No response_

### Quasar info output

```Shell
-
```


### Relevant log output

```Shell
-
```


### Additional context

-

Hi, i've faced the same problem here:

![image](https://user-images.githubusercontent.com/1966285/168934307-7583dc2f-3068-4cba-b113-31acf70a6e14.png)

```console
√ Install project dependencies? (recommended) » Yes, use npm

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: celebra-ui@0.0.1
npm ERR! Found: eslint-plugin-n@14.0.0
npm ERR! node_modules/eslint-plugin-n
npm ERR!   dev eslint-plugin-n@"^14.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-n@"^15.0.0" from eslint-config-standard@17.0.0
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"^17.0.0-1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\almei\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\almei\AppData\Local\npm-cache\_logs\2022-05-18T00_40_03_987Z-debug-0.log

 npm FAILED...
```

Yeah sorry I see I was quite vague.

I went to quasar.dev and followed the steps for installing a vite based @quasar/cli install

```bash
npm i -g @quasar/cli
npm init quasar



 .d88888b.
d88P" "Y88b
888     888
888     888 888  888  8888b.  .d8888b   8888b.  888d888
888     888 888  888     "88b 88K          "88b 888P"
888 Y8b 888 888  888 .d888888 "Y8888b. .d888888 888
Y88b.Y8b88P Y88b 888 888  888      X88 888  888 888
 "Y888888"   "Y88888 "Y888888  88888P' "Y888888 888
       Y8b

✔ What would you like to build? › App with Quasar CLI, let's go!
✔ Project folder: … quasar-demo
✔ Pick Quasar version: › Quasar v2 (Vue 3 | latest and greatest)
✔ Pick script type: › Javascript
✔ Pick Quasar App CLI variant: › Quasar App CLI with Vite (BETA stage)
✔ Package name: … quasar-demo
✔ Project product name: (must start with letter if building mobile apps) … Quasar App
✔ Project description: … A Quasar Project
✔ Author: … mixmix
✔ Pick your CSS preprocessor: › Sass with SCSS syntax
✔ Check the features needed for your project: › ESLint, State Management (Pinia), Vue-i18n
✔ Pick an ESLint preset: › Standard

 Quasar • Generating files...

 - README.md
 - .editorconfig
 - .gitignore
 - jsconfig.json
 - package.json
 - index.html
 - postcss.config.js
 - quasar.config.js
 - .vscode/extensions.json
 - .vscode/settings.json
 - public/favicon.ico
 - src/App.vue
 - public/icons/favicon-128x128.png
 - public/icons/favicon-16x16.png
 - public/icons/favicon-32x32.png
 - public/icons/favicon-96x96.png
 - src/assets/quasar-logo-vertical.svg
 - src/boot/.gitkeep
 - src/components/EssentialLink.vue
 - src/layouts/MainLayout.vue
 - src/pages/ErrorNotFound.vue
 - src/pages/IndexPage.vue
 - src/router/index.js
 - src/router/routes.js
 - src/css/app.scss
 - src/css/quasar.variables.scss
 - src/boot/i18n.js
 - src/i18n/index.js
 - src/i18n/en-US/index.js
 - .eslintignore
 - .eslintrc.js
 - src/stores/example-store.js
 - src/stores/index.js
 - src/stores/store-flag.d.ts

 Quasar •  SUCCESS  • The project has been scaffolded

✔ Install project dependencies? (recommended) › Yes, use npm

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: quasar-demo@0.0.1
npm ERR! Found: eslint-plugin-n@14.0.0
npm ERR! node_modules/eslint-plugin-n
npm ERR!   dev eslint-plugin-n@"^14.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-n@"^15.0.0" from eslint-config-standard@17.0.0
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"^17.0.0-1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/myhome/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/myhome/.npm/_logs/2022-05-18T10_36_24_182Z-debug-0.log

 npm FAILED...

 Quasar • ⚠️  Could not auto install dependencies. Probably a temporary npm registry issue?

To get started:

  cd quasar-demo
  yarn #or: npm install
  yarn lint --fix # or: npm run lint -- --fix
  quasar dev # or: yarn quasar dev # or: npx quasar dev
```

--- a/app-vite/package.json
+++ b/app-vite/package.json
@@ -86,7 +86,7 @@
     "electron-builder": "22.14.13",
     "electron-packager": "^15.3.0",
     "eslint": "^8.11.0",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^15.0.0",
     "pinia": "^2.0.0",
     "quasar": "^2.0.4",
     "ts-essentials": "^9.1.2",

--- a/app-webpack/package.json
+++ b/app-webpack/package.json
@@ -116,7 +116,7 @@
     "electron-builder": "22.14.13",
     "electron-packager": "^15.3.0",
     "eslint": "^8.11.0",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^15.0.0",
     "pinia": "^2.0.0",
     "quasar": "^2.0.4",
     "ts-essentials": "^9.1.2",

--- a/create-quasar/templates/app/quasar-v2/js-vite/BASE/_package.json
+++ b/create-quasar/templates/app/quasar-v2/js-vite/BASE/_package.json
@@ -26,9 +26,9 @@
     "eslint": "^8.10.0",
     "eslint-plugin-vue": "^8.5.0",
     <% if (lintConfig === 'standard') { %>
-    "eslint-config-standard": "^17.0.0-1",
+    "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.19.1",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^15.0.0",
     "eslint-plugin-promise": "^6.0.0",
     <% } else if (lintConfig === 'airbnb') { %>
     "eslint-config-airbnb-base": "^15.0.0",

--- a/create-quasar/templates/app/quasar-v2/js-webpack/BASE/_package.json
+++ b/create-quasar/templates/app/quasar-v2/js-webpack/BASE/_package.json
@@ -29,9 +29,9 @@
     "eslint-plugin-vue": "^8.5.0",
     "eslint-webpack-plugin": "^3.1.1",
     <% if (lintConfig === 'standard') { %>
-    "eslint-config-standard": "^17.0.0-1",
+    "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.19.1",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^15.0.0",
     "eslint-plugin-promise": "^6.0.0",
     <% } else if (lintConfig === 'airbnb') { %>
     "eslint-config-airbnb-base": "^15.0.0",

--- a/create-quasar/templates/app/quasar-v2/ts-vite/BASE/_package.json
+++ b/create-quasar/templates/app/quasar-v2/ts-vite/BASE/_package.json
@@ -29,9 +29,9 @@
     "eslint": "^8.10.0",
     "eslint-plugin-vue": "^8.5.0",
     <% if (lintConfig === 'standard') { %>
-    "eslint-config-standard": "^17.0.0-1",
+    "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.19.1",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^15.0.0",
     "eslint-plugin-promise": "^6.0.0",
     <% } else if (lintConfig === 'airbnb') { %>
     "eslint-config-airbnb-base": "^15.0.0",

--- a/create-quasar/templates/app/quasar-v2/ts-webpack/BASE/_package.json
+++ b/create-quasar/templates/app/quasar-v2/ts-webpack/BASE/_package.json
@@ -30,9 +30,9 @@
     "eslint": "^8.10.0",
     "eslint-plugin-vue": "^8.5.0",
     <% if (lintConfig === 'standard') { %>
-    "eslint-config-standard": "^17.0.0-1",
+    "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.19.1",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^15.0.0",
     "eslint-plugin-promise": "^6.0.0",
     <% } else if (lintConfig === 'airbnb') { %>
     "eslint-config-airbnb-base": "^15.0.0",

--- a/docs/package.json
+++ b/docs/package.json
@@ -23,7 +23,7 @@
     "@quasar/app-vite": "^1.0.0-beta.9",
     "autoprefixer": "^10.4.2",
     "eslint": "^8.10.0",
-    "eslint-config-standard": "^17.0.0-1",
+    "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.19.1",
     "eslint-plugin-n": "^15.1.0",
     "eslint-plugin-promise": "^6.0.0",

